packages/auth/src/index.ts

```diff
@@ -325,7 +325,7 @@ const callWithRetry: CallableFunction = async (
   try {
     return await fn()
   } catch (e) {
-    if (depth > 7) {
```

Comment:
I think we can remove this one and the backend one this is the configuration for the migration files. 
We are just worried about the ASE in this change, if I understand correctly

Reply:
yes, removed